### PR TITLE
XWiki 9.8.1. is released

### PR DESCRIPTION
**Pull request by gdelhumeau**

### CODE REVIEW

**Comment by tianon** [comment]
cc @vmassol (this look good to you?)

<details>
<summary>Diff:</summary>

```diff
diff --git a/_bashbrew-list b/_bashbrew-list
index b6fa78c..187b1a0 100644
--- a/_bashbrew-list
+++ b/_bashbrew-list
@@ -10,6 +10,7 @@ xwiki:9-mysql-tomcat
 xwiki:9-postgres-tomcat
 xwiki:9.8
 xwiki:9.8-postgres-tomcat
+xwiki:9.8.1
 xwiki:latest
 xwiki:lts
 xwiki:lts-mysql
diff --git a/xwiki_stable-postgres/Dockerfile b/xwiki_stable-postgres/Dockerfile
index 5dc282c..1dd58f2 100644
--- a/xwiki_stable-postgres/Dockerfile
+++ b/xwiki_stable-postgres/Dockerfile
@@ -44,9 +44,9 @@ RUN apt-get update && \
 # Install XWiki as the ROOT webapp context in Tomcat
 # Create the Tomcat temporary directory
 # Configure the XWiki permanent directory
-ENV XWIKI_VERSION=9.8
+ENV XWIKI_VERSION=9.8.1
 ENV XWIKI_URL_PREFIX "http://maven.xwiki.org/releases/org/xwiki/platform/xwiki-platform-distribution-war/${XWIKI_VERSION}"
-ENV XWIKI_DOWNLOAD_SHA256 f416d4fccf6f766c902483b2cbc56236924c62ba2f5513128026c65f8d163ae2
+ENV XWIKI_DOWNLOAD_SHA256 26f55d585ed1b05cf730af585ab1cd0751e2b09770e2ab27c11e3b55027b9733
 RUN rm -rf /usr/local/tomcat/webapps/* && \
   mkdir -p /usr/local/tomcat/temp && \
   mkdir -p /usr/local/xwiki/data && \
diff --git a/xwiki_stable/Dockerfile b/xwiki_stable/Dockerfile
index 608b358..1fe79a1 100644
--- a/xwiki_stable/Dockerfile
+++ b/xwiki_stable/Dockerfile
@@ -44,9 +44,9 @@ RUN apt-get update && \
 # Install XWiki as the ROOT webapp context in Tomcat
 # Create the Tomcat temporary directory
 # Configure the XWiki permanent directory
-ENV XWIKI_VERSION=9.8
+ENV XWIKI_VERSION=9.8.1
 ENV XWIKI_URL_PREFIX "http://maven.xwiki.org/releases/org/xwiki/platform/xwiki-platform-distribution-war/${XWIKI_VERSION}"
-ENV XWIKI_DOWNLOAD_SHA256 f416d4fccf6f766c902483b2cbc56236924c62ba2f5513128026c65f8d163ae2
+ENV XWIKI_DOWNLOAD_SHA256 26f55d585ed1b05cf730af585ab1cd0751e2b09770e2ab27c11e3b55027b9733
 RUN rm -rf /usr/local/tomcat/webapps/* && \
   mkdir -p /usr/local/tomcat/temp && \
   mkdir -p /usr/local/xwiki/data && \
````

</details>

**Comment by tianon** [comment]
LGTM (just waiting for confirmation from @vmassol)

Build test of #3541; 9ca37144 (`xwiki`):

```console

$ bashbrew build xwiki:8
Using bashbrew/cache:8c60f454a4424263ee259da38fdcfa82b0505d915d62f10d6898c4a9b052c808 (xwiki:8)
Tagging xwiki:8
Tagging xwiki:8.4
Tagging xwiki:8.4.5
Tagging xwiki:8-mysql-tomcat
Tagging xwiki:mysql-tomcat
Tagging xwiki:lts-mysql-tomcat
Tagging xwiki:lts-mysql
Tagging xwiki:lts
Tagging xwiki:latest

$ test/run.sh xwiki:8
testing xwiki:8
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build xwiki:9
Using bashbrew/cache:6d99e65fcac2cd29f3a31656086c70f693ab00d65d6144ae05a4f386fbf0e19e (xwiki:9)
Tagging xwiki:9
Tagging xwiki:9.8
Tagging xwiki:9.8.1
Tagging xwiki:9-mysql-tomcat
Tagging xwiki:stable-mysql-tomcat
Tagging xwiki:stable-mysql
Tagging xwiki:stable

$ test/run.sh xwiki:9
testing xwiki:9
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build xwiki:8-postgres-tomcat
Using bashbrew/cache:99db84121ce352d46cddfbb40869a0c6d561bad4c62ffda4402f4bfb601a0ff2 (xwiki:8-postgres-tomcat)
Tagging xwiki:8-postgres-tomcat
Tagging xwiki:8.4-postgres-tomcat
Tagging xwiki:8.4.5-postgres-tomcat
Tagging xwiki:postgres-tomcat
Tagging xwiki:lts-postgres-tomcat
Tagging xwiki:lts-postgres

$ test/run.sh xwiki:8-postgres-tomcat
testing xwiki:8-postgres-tomcat
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build xwiki:9-postgres-tomcat
Using bashbrew/cache:19cf323c37743e23e3af09477f6184b335e5da529a762cd7047ba16f7a5612ac (xwiki:9-postgres-tomcat)
Tagging xwiki:9-postgres-tomcat
Tagging xwiki:9.8-postgres-tomcat
Tagging xwiki:stable-postgres-tomcat
Tagging xwiki:stable-postgres

$ test/run.sh xwiki:9-postgres-tomcat
testing xwiki:9-postgres-tomcat
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed

```

**Comment by vmassol** [comment]
Seems good to me, thanks!

**Comment by vmassol** [comment]
Actually wait, adding a small change

File: library/xwiki [hunk 1]
```diff
@@ -5,14 +5,14 @@ Tags: 8, 8.4, 8.4.5, 8-mysql-tomcat, mysql-tomcat, lts-mysql-tomcat, lts-mysql,
...
-GitCommit: 02e12b746e3029ab4edf0ecedb2d4a1fe3f623c8
+Tags: 9, 9.8, 9.8.1, 9-mysql-tomcat, stable-mysql-tomcat, stable-mysql, stable
+GitCommit: 088aaf39de290ece849eabace996b21f3b99d954
 Directory: 9/mysql-tomcat
 
 Tags: 8-postgres-tomcat, 8.4-postgres-tomcat, 8.4.5-postgres-tomcat, postgres-tomcat, lts-postgres-tomcat, lts-postgres
 GitCommit: fbc0fe3cbc07119c45ae8ea97c7e021d41e89b0a
 Directory: 8/postgres-tomcat
 
 Tags: 9-postgres-tomcat, 9.8-postgres-tomcat, stable-postgres-tomcat, stable-postgres
```
Comment (vmassol) [inline library/xwiki]: I believe we're missing the `9.8.1-postgres-tomcat` tag here.

**Comment by vmassol** [comment]
Seems I can't make the change on my side, waiting for @gdelhumeau to fix it.

**Comment by tianon** [comment]
@vmassol if you'd rather not wait for @gdelhumeau, feel free to make a new PR and we will simply merge whichever one makes the appropriate change sooner :+1: (and close the other)

**Comment by vmassol** [comment]
@tianon Thanks. I've created https://github.com/docker-library/official-images/pull/3553

**Comment by tianon** [comment]
Closing in favor of #3553; thanks @gdelhumeau! :heart:

**Pull request closed without merging**